src/app/public/modules/text-editor/services/text-editor-adapter.service.ts

```diff
@@ -538,4 +525,13 @@ export class SkyTextEditorAdapterService {
       .replace(/&amp;/g, '&');
   }
 
+  private toggleEditorAbility(id: string, focusableChildren: HTMLElement[], textEditorNativeElement: any, disabled: boolean): void {
```

Comment:
`Ability` is vague... would you mind making this more explicit by using `disabled` ? Something like `toggleEditorDisabled` or even more explicit... `setEditorDisabled` - This would help developers infer what the boolean is doing without reading the argument names. For example:

`setEditorDisabled(true)` = I'm turning on the disabled state

`setEditorDisabled(false)` = I'm turning off the disabled state

Reply:
I like that format. I'll update!

Reply:
fixed in commit: ["update function name"](https://github.com/blackbaud/skyux-text-editor/pull/21/commits/3ddac164875a37c14470ae2172fa63859af0de63)